products/hashicorp-vault.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Vault
+permalink: /hashicorp-vault
```

Comment:
We can use /vault as an alternate URL to provide an easy to use shortlink (endoflife.date/vault). In case this does become ambiguous in the future, we can always remove the redirect or add a second page)

Reply:
I applied the suggested change